en/docs/includes/deploy/steps-to-deploy-apim-in-a-distributed-setup-with-tm-separation.md

```diff
@@ -86,7 +86,7 @@ Follow the instructions given below to configure the Gateway node so that it can
     !!! Info
         Event hub configuration is used to retrieve Gateway artifacts. Using `event_listening_endpoints`, the Gateway will create a JMS connection with the event hub that is then used to subscribe for API/Application/Subscription and Key Manager operations-related events. The `service_url` points to the internal API that resides in the event hub that is used to pull artifacts and information from the database.
 
-    * **Connecting the Gateway to the Traffic Manager node**:
+    - **Connecting the Gateway to the Traffic Manager node**:
```

Comment:
```suggestion
    * **Connecting the Gateway to the Traffic Manager node**:
```

Reply:
Seems we don't have a `steps-to-deploy-apim-in-a-distributed-setup-with-tm-separation.md` directed from the mkdocs.yaml. We don't need to add this change.